src/pkg/bigbang/bigbang.go

```diff
@@ -0,0 +1,500 @@
+	if err != nil {
+		return fmt.Errorf("unable to get flux kustomization file: %w", err)
+	}
+
+	var fluxKustomize kustypes.Kustomization
+
+	goyaml.Unmarshal(kustFile, &fluxKustomize)
+
+	if FluxImages[version] == nil {
+		FluxImages[version] = make([]string, 0)
+	}
+
+	for _, image := range fluxKustomize.Images {
+		FluxImages[version] = utils.Unique(append(FluxImages[version], fmt.Sprintf("%s:%s", image.NewName, image.NewTag)))
+	}
+	return nil
+}
+
+// importBigbangImageList  populates an internal map of the images
+// needed for the version of flux that is defined within the provided version
+// of BigBang
+func importBigbangImageList(version string) error {
+	repo1, err := gitlab.NewClient("", gitlab.WithBaseURL("https://repo1.dso.mil/api/v4"))
+	if err != nil {
+		return fmt.Errorf("unable to create gitlab client: %w", err)
+	}
+
+	release, _, err := repo1.Releases.GetRelease("2872", version, nil)
+	if err != nil {
+		return fmt.Errorf("unable to get release: %w", err)
+	}
+
+	var imagesUrl string
+
+	for _, link := range release.Assets.Links {
+		if link.Name == "package-images.yaml" {
+			imagesUrl = link.URL
+			break
+		}
+	}
+
+	resp, err := http.Get(imagesUrl)
+	if err != nil {
+		return fmt.Errorf("unable to get package-images.yaml: %w", err)
+	}
+	defer resp.Body.Close()
+
+	imageYamlBytes, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return fmt.Errorf("unable to read package-images.yaml: %w", err)
+	}
+
+	var imageYaml bbImageYaml
+
+	goyaml.Unmarshal(imageYamlBytes, &imageYaml)
+
+	for chartName, versionAndImages := range imageYaml.ChartList {
+		fixedChartName := bbChartNameFix(chartName)
+
+		if BigBangImages[fixedChartName] == nil {
+			BigBangImages[fixedChartName] = make(map[string][]string)
+		}
+
+		BigBangImages[fixedChartName][versionAndImages.Version] =
+			utils.Unique(
+				append(BigBangImages[fixedChartName][versionAndImages.Version], versionAndImages.Images...))
+	}
+	return nil
+}
+
+func printObject(filename string, o runtime.Object) error {
+	return nil
+
+}
+
+func bbChartNameFix(chartName string) string {
+	switch chartName {
+	case "istio":
+		return "istio-controlplane"
+	case "istiooperator":
+		return "istio-operator"
+	case "kyvernopolicies":
+		return "kyverno-policies"
+	case "gatekeeper":
+		return "policy"
+	case "clusterAuditor":
+		return "cluster-auditor"
+	case "eckoperator":
+		return "eck-operator"
+	case "logging":
+		return "elasticsearch-kibana"
+	case "metricsServer":
+		return "metrics-server"
+	default:
+		return chartName
+	}
+}
+
+func GetFluxManifest(version string) types.ZarfManifest {
+	return types.ZarfManifest{
+		Name:      "flux-system",
+		Namespace: "flux-system",
+		Kustomizations: []string{
+			fmt.Sprintf("https://repo1.dso.mil/platform-one/big-bang/bigbang.git//base/flux?ref=%v", version),
+		},
+	}
+}
+
+func GetBigBangManifests(manifestDir string, component types.ZarfComponent) (types.ZarfManifest, error) {
```

Comment:
https://github.com/defenseunicorns/zarf/issues/1092

This is essentially the wrapper I'd like to make an option for any helm chart via this issue